lib/src/Pos/Launcher/Configuration.hs

```diff
@@ -133,7 +172,8 @@ withConfigurationsM logName mAssetLockPath cfo fn act = do
         withSscConfiguration (ccSsc cfg) $
         withDlgConfiguration (ccDlg cfg) $
         withBlockConfiguration (ccBlock cfg) $
-        withNodeConfiguration (ccNode cfg) $ \ pm -> act pm (addAssetLock assetLock $ ccTxp cfg) (ccNtp cfg)
+        withNodeConfiguration (ccNode cfg) $ \ pm ->
```

Comment:
The space between `\` and argument's name? Is it a good decision?

Reply:
That's how it was previously :man_shrugging: I can delete the space since the line is touched anyway.